packages/semantic-model/test/api-spec.ts

```diff
@@ -233,41 +235,45 @@ context("The ui5-language-assistant semantic model package API", () => {
 
   function createModelConsistencyTests(version: TestModelVersion): void {
     describe(`Model generated from ${version}`, () => {
-      it(`is created successfully in strict mode`, () => {
-        const model = generateModel(version);
+      before(async function() {
+        this.timeout(GEN_MODEL_TIMEOUT);
+        await downloadLibraries(version);
+      });
+
+      it(`is created successfully in strict mode`, async () => {
+        const model = await generateModel(version, false);
         expect(model).to.exist;
       });
-      it(`has correct parent on root symbols`, () => {
-        const model = generateModel(version);
+      it(`has correct parent on root symbols`, async () => {
+        const model = await generateModel(version, false);
```

Comment:
should the model generation be extracted to `before` clause?

Reply:
The first test tests it... I could add another describe for the other 3 tests

Reply:
your choice